src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefReadonlyReturnTests.cs

```diff
@@ -28,7 +28,7 @@ class Program
 }
 ";
 
-            var comp = CompileAndVerify(text, parseOptions: TestOptions.Regular);
+            var comp = CompileAndVerify(text, verify: false, parseOptions: TestOptions.Regular);
```

Comment:
After chatting with Vlad, our approach so far is to disable it in cases where we know PEVerify is failing because of that. I will do a quick pass on the tests touched by this PR (and possibly others concerning the same feature) to check if we can enable it on some, but it is pretty much a hit and miss in this case.

---
In reply to: [115350911](https://github.com/dotnet/roslyn/pull/18715#discussion_r115350911) [](ancestors = 115350911,115337794)